modules/arm_plugin/armplg_build.sh

```diff
@@ -109,15 +110,13 @@ cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_LIST=imgcodecs,videoio,highgui,gapi
       -D OPENCV_OTHER_INSTALL_PATH=etc \
       -D OPENCV_LICENSES_INSTALL_PATH=etc/licenses \
       -D ENABLE_CXX11=ON \
-      -D CMAKE_INSTALL_PREFIX=install \
       -DCMAKE_TOOLCHAIN_FILE="$OPENVINO_HOME/cmake/$TOOLCHAIN_DEFS" \
-      -DCMAKE_STAGING_PREFIX=$STAGING_DIR/opencv \
       -DWITH_GTK_2_X=OFF \
       -DOPENCV_ENABLE_PKG_CONFIG=ON \
       -DPKG_CONFIG_EXECUTABLE=/usr/bin/${ARCH_NAME}-pkg-config \
       $OPENCV_HOME && \
```

Comment:
in general, it's an awful script. Do we really need a great deal of option to build OpenCV for arm?

```cmake
cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_LIST=imgcodecs,videoio,highgui,gapi,python3 \
      -DBUILD_opencv_python2=OFF -DBUILD_opencv_python3=ON -DOPENCV_SKIP_PYTHON_LOADER=ON \
      -DPYTHON3_LIMITED_API=ON -DPYTHON3_PACKAGES_PATH=$STAGING_DIR/opencv/python \
      -DPYTHON3_INCLUDE_PATH=/usr/include/${PYTHONVER} \
      -DPYTHON3_LIBRARIES=/usr/lib/$ARCH_NAME/lib${PYTHONVER}m.so \
      -DPYTHON3_NUMPY_INCLUDE_DIRS=/usr/lib/python3/dist-packages/numpy/core/include \
      -D CMAKE_USE_RELATIVE_PATHS=ON \
      -D CMAKE_SKIP_INSTALL_RPATH=ON \
      -D OPENCV_SKIP_PKGCONFIG_GENERATION=ON \
      -D OPENCV_SKIP_PYTHON_LOADER=ON \
      -D OPENCV_SKIP_CMAKE_ROOT_CONFIG=ON \
      -D OPENCV_BIN_INSTALL_PATH=bin \
      -D OPENCV_INCLUDE_INSTALL_PATH=include \
      -D OPENCV_LIB_INSTALL_PATH=lib \
      -D OPENCV_CONFIG_INSTALL_PATH=cmake \
      -D OPENCV_3P_LIB_INSTALL_PATH=3rdparty \
      -D OPENCV_SAMPLES_SRC_INSTALL_PATH=samples \
      -D OPENCV_DOC_INSTALL_PATH=doc \
      -D OPENCV_OTHER_INSTALL_PATH=etc \
      -D OPENCV_LICENSES_INSTALL_PATH=etc/licenses \
      -D ENABLE_CXX11=ON \
      -DCMAKE_TOOLCHAIN_FILE="$OPENVINO_HOME/cmake/$TOOLCHAIN_DEFS" \
      -DWITH_GTK_2_X=OFF \
      -DOPENCV_ENABLE_PKG_CONFIG=ON \
      -DPKG_CONFIG_EXECUTABLE=/usr/bin/${ARCH_NAME}-pkg-config \
      $OPENCV_HOME && \
```
CC @alalek

Reply:
Do you suggest to download already pre-built OpenCV package?

Reply:
> INSTALL_PATH

these options are required to follow OpenVINO's layout

These can be dropped:
- OPENCV_SKIP_CMAKE_ROOT_CONFIG (Win32 only)
- ENABLE_CXX11 (OpenCV 4.x requires/enables C++11 by default)

`OPENCV_SKIP_PYTHON_LOADER` is used twice. Also for proper support of G-API Python API we need `OPENCV_SKIP_PYTHON_LOADER=OFF` (OpenCV default value)

Reply:
Removed `OPENCV_SKIP_CMAKE_ROOT_CONFIG` and `ENABLE_CXX11`.
Duplication of `OPENCV_SKIP_PYTHON_LOADER` is removed. `OPENCV_SKIP_PYTHON_LOADER` value changed to `OFF`.